dds_web/utils.py

```diff
@@ -598,3 +598,16 @@ def block_if_maintenance(user=None):
         else:
             if user.role != "Super Admin":
                 raise MaintenanceOngoingException()
+
+
+def convert_from_bytes(num_bytes: int, convert_to: str, base: int = 10):
+    """Convert number of bytes to KB, MB, GB, TB, PB."""
+    # Define how many thousands in formats
+    magnitudes: typing.Dict = {"KB": 1, "MB": 2, "GB": 3, "TB": 4, "PB": 5}
+
+    # Default is 1000 but base 2 changes to 1024
+    base_magnitude: int = 1000
+    if base == 2:
+        base_magnitude = 1024
+
+    return num_bytes / (base_magnitude ** magnitudes[convert_to])
```

Comment:
I had some idea for how to use this but now it's unused for some reason and I'm unsure what the goal was. Maybe I'll remember tomorrow.